tests/test_ssl_transport_close.py

```diff
@@ -0,0 +1,83 @@
+from aiohttp import web
+import aiohttp
+import asyncio
+import warnings
+
+
+async def handle_root(request):
+    """
+    idle 2 seconds so service is restarted meanwhile, forcing connection_lost
+    """
+    await asyncio.sleep(2)
+    return web.Response(text='', status=200)
+
+
+async def _prepare(ssl_ctx, client_ssl_ctx):
+    app = web.Application()
+    app.router.add_get('/', handle_root)
+    runner = web.AppRunner(app)
```

Comment:
It's better not to reimplement the server initialization. Just use our native [`aiohttp_server` fixture](https://github.com/aio-libs/aiohttp/blob/f8df85d/aiohttp/pytest_plugin.py#L261). It at least will get you a server with an app runner set up and you can probably extend it with a tcp site on top.